inference/core/models/roboflow.py

```diff
@@ -416,15 +416,17 @@ def letterbox_image(
         Returns:
             Image.Image: PIL image of dimensions `size` containing the original image plus any required padding
         """
-        iw, ih = image.size
-        w, h = size
+        ih, iw = image.shape[0:2]
+        h, w = size
         scale = min(w / iw, h / ih)
         nw = int(iw * scale)
         nh = int(ih * scale)
 
-        image = image.resize((nw, nh), Image.BICUBIC)
-        new_image = Image.new("RGB", size, (114, 114, 114))
-        new_image.paste(image, ((w - nw) // 2, (h - nh) // 2))
+        image = cv2.resize(image, (nh, nw), cv2.INTER_CUBIC)
+        new_image = np.ones((size,) + 3).fill(c)
```

Comment:
May want to use `np.full` so you odn't have to do two passes across the new image

Reply:
Nice! Will do!